learn/wme-backup-restore/wme-backup-restore-introduction.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "WME-Backup-Restore"
+id: "wme-backup-restore-introduction"
+sidebar_label: "Introduction"
+---
+---
+## Introduction
+
+WaveMaker uses separate dedicated directory for storing data, which is easy to take backup and restore
```

Comment:
WaveMaker Platform Stores it's state into disk(specific directories as mentioned in the below images). WaveMaker Platform administrtors can take backups of those disk/directories and can restore to any previous state.

In this document we will learn how to take backups and restore in various infrastructure providers.